examples/batch/batch.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+"""
+Example of running Batch script with analysis-runner.
+"""
+import os
+import hailtop.batch as hb
+import click
+
+DATASET = os.getenv('DATASET')
+BUCKET = os.getenv('HAIL_BUCKET')
+BILLING_PROJECT = os.getenv('HAIL_BILLING_PROJECT')
+ACCESS_LEVEL = os.getenv('ACCESS_LEVEL')
+
+
+@click.command()
+@click.option('--cram', 'cram_path', 'Input CRAM or BAM file')
+@click.option('--output', 'output_path', 'Output path to write the result')
```

Comment:
Can we build this argument from environment variables, eg:

```python
# or something like this
output_filename = os.path.basename(cram_path) + '-processed.cram'
output_prefix = f'gs://{BUCKET}/{os.getenv("OUTPUT")}/{output_filename}'
```

Reply:
Sure, changed it to use the env variables. Though writing permanent results into the `HAIL_BUCKET` might not be a good idea in practice. But doesn't matter for an example

Reply:
Can you tack on `-tmp` on the end of the {BUCKET}?